src/nodes/player.gd

```diff
@@ -23,6 +27,20 @@ func _ready():
 
 
 func _physics_process(delta):
+	move()
+	
+	if wants_to_shoot(): shoot()
+
+
+func _process(delta):
```

Comment:
what's the difference between _process and _physics_process?

Reply:
`void _physics_process(delta: float) virtual`

Called during the physics processing step of the main loop. Physics processing means that the frame rate is synced to the physics, i.e. the delta variable should be constant. delta is in seconds.


`void _process(delta: float) virtual`

Called during the processing step of the main loop. Processing happens at every frame and as fast as possible, so the delta time since the previous frame is not constant. delta is in seconds.

Reply:
thank you :kissing_heart: